PhpcrMigration/Infrastructure/Repository/EntityRepository.php

```diff
@@ -253,4 +254,9 @@ private function getSequences(): array
 
         return $this->sequences = $sequences;
     }
+
+    public function reset(): void
+    {
+        unset($this->sequences);
```

Comment:
unset would remove the prop technical we should just reset it to the default value.

```suggestion
        $this->sequences = [];
```

Reply:
Yes but an empty array is a valid value, so to store it, we have to unset it, so that it is not initialized anymore

Reply:
> Yes but an empty array is a valid value

Values are initialized with null so we should set to null:

```suggestion
        $this->sequences = null;
```

Reply:
No it is not automatically initialized with null, if it is not nullable and you try to access it, php throws an exception 😅 

But I can refactor it to be nullable if you insist on it

Reply:
done :)